frontend/lib/norent/components/social-icons.tsx

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+import { StaticImage } from "../../ui/static-image";
+import { getImageSrc } from "../homepage";
+import { OutboundLink } from "../../analytics/google-analytics";
+import { getGlobalAppServerInfo } from "../../app-context";
+
+/**
+ * Links to JustFix.nyc's main social media pages.
+ */
+const socialMediaPageLinks = [
+  { name: "twitter", url: "https://twitter.com/JustFixNYC" },
+  { name: "facebook", url: "https://www.facebook.com/JustFixNYC/" },
+  { name: "instagram", url: "https://www.instagram.com/justfixnyc/" },
+];
+
+const prefilledTwitterCopy =
+  "No idea how you'll pay rent this month? Tell your landlord with norent.org from @JustFixNYC. " +
+  "This free tool sends a certified letter informing them of your protections. " +
+  "Join the #cancelrent movement at norent.org.";
+const prefilledEmailSubject =
+  "Just used JustFix.nyc's new free tool to tell my landlord I can't pay rent";
+const prefilledEmailBody =
+  "I used www.norent.org to tell my landlord that I'm unable to pay May's rent. " +
```

Comment:
Should we actually mention "May" here?  Seems like it will get out of date pretty fast, maybe it should say "this month's rent" instead so it's more evergreen?

Reply:
yeah, good catch!!